instrumentation/netty/netty-4.0/javaagent/src/test/groovy/ChannelPipelineTest.groovy

```diff
@@ -89,6 +89,51 @@ class ChannelPipelineTest extends AgentInstrumentationSpecification {
     "by name"     | { pipeline, oldName, oldHandler, newName, newHandler -> pipeline.replace(oldName, newName, newHandler) }
   }
 
+  // regression test for https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/4056
+  def "should addAfter and removeLast handler #desc"() {
+    setup:
+    def channel = new EmbeddedChannel(new NoopChannelHandler())
+    def channelPipeline = new DefaultChannelPipeline(channel)
+    def httpHandler = new HttpClientCodec()
+
+    expect: "no handlers initially"
+    channelPipeline.size() == 0
+
+    when:
+    channelPipeline.addLast("http", httpHandler)
+    channelPipeline.first() == httpHandler
+    // our handler was also added
+    channelPipeline.last().getClass() == HttpClientTracingHandler
```

Comment:
These 2 assertions are already present in the `then:` section.

Reply:
Same applies to the other test.

Reply:
thanks, should I remove then in this pr?

Reply:
Yes, please 👍